### PR TITLE
Feature/135 apply functions from datatools

### PR DESCRIPTION
**Pull request by jan-abel-inwt**

### CODE REVIEW

**Review by arunge — CHANGES_REQUESTED** [review]
Could you please merge the changes from `main` into this branch? Everything else looks ok. :+1: 

image

**Comment by jan-abel-inwt** [comment]
> Could you please merge the changes from `main` into this branch? Everything else looks ok. +1 https://github.com/Pandora-IsoMemo/iso-app/network image

Done.

File: R/01-isomemoAPI.R [hunk 1]
```diff
@@ -1,93 +0,0 @@
...
-  colToFill <- mapping$shiny[!(mapping$shiny %in% names(data))]
-  data[colToFill] <- NA
-  data
-}
-
-#' getMappingTable
-#'
-#' @export
-getMappingTable <- function() {
-  getMappingAPI()
```
Comment (arunge) [inline R/01-isomemoAPI.R:67]: analog to `getDatabaseList`

File: R/01-isomemoAPI.R [hunk 2]
```diff
@@ -1,93 +0,0 @@
...
-
-#' getRemoteData
-#'
-#' @param db database
-#'
-#' @export
-getRemoteData <- function(db) {
-  if (is.null(db)) return(NULL)
-
-  isoData <- getRemoteDataAPI(db = db)
```
Comment (arunge) [inline R/01-isomemoAPI.R:78]: analog to `getDatabaseList`

**Review by arunge — CHANGES_REQUESTED** [review]
@jan-abel-inwt Please let us discuss the issue tomorrow (and rethink the case together) before you start changing something

**Comment by arunge** [comment]
As just discussed, forget the above, just add the lines 
```R
if (Sys.getenv("API_BASE_URL") != "") {
    apiBaseURL <- Sys.getenv("API_BASE_URL")
  } else {
    apiBaseURL <- Sys.getenv("API_BASE_URL_DEFAULT")
}
```
instead of 
```R
apiBaseURL <- Sys.getenv("API_BASE_URL")
```
to the DataTools function! :slightly_smiling_face: 

Then the error should be fixed.

**Review by arunge — DISMISSED** [review]
With the new version 23.04.3.1 of DataTools the issue is solved.

**Review by arunge — APPROVED** [review]
:+1:

**Pull request merged**